config/defaults.go

```diff
@@ -47,25 +47,31 @@ func getDefaultPeerRpcConfig() *PeerRpcConfig {
 	}
 }
 
-func getPeersConfig() []*PeerConfig {
-	return []*PeerConfig{
-		{
-			Host: "localhost",
-			Port: &PeerPortConfig{
-				P2P: 18333,
-				ZMQ: 28332,
+func getPeersConfig() *BroadcastingConfig {
```

Comment:
Rename to `getBroadcastingConfig`
```suggestion
func getBroadcastingConfig() *BroadcastingConfig {
```